prerendering.bs

```diff
@@ -119,6 +120,9 @@ spec: web-audio; urlPrefix: https://webaudio.github.io/web-audio-api/
     text: running; url: dom-audiocontextstate-running
     text: queue a control message; url: queuing
     text: [[pending resume promises]]; url: dom-audiocontext-pending-resume-promises-slot
+spec: webappsec-clear-site-data; urlPrefix: https://w3c.github.io/webappsec-clear-site-data/
+  type: dfn
+    text: Clear cache for origin; url: clear-cache
```

Comment:
I think you can remove this line as long as you use `[$clear cache for origin$]` instead of `[=clear cache for origin=]`.

This is because they have, for some reason, chosen to define this algorithm as an "abstract operation" instead of a normal definition.

Reply:
You can find all publicly-exported definitions in https://dontcallmedom.github.io/webdex/, so e.g. I was able to find https://dontcallmedom.github.io/webdex/c.html#clear%20cache%20for%20origin%40%40clear-site-data%25%25abstract-op

We have a large anchors block because we are specifically poking at a lot of non-public definitions, due to our nature as a spec full of patches. "Normal" specs just get magic autolinking via the webdex database.